ddcommon/src/connector/mod.rs

```diff
@@ -96,6 +96,8 @@ mod https {
     /// sometimes this is done as a side-effect of other operations, but we need to ensure it
     /// happens here.  On non-unix platforms, ddcommon uses `ring` instead, which handles this
     /// at rustls initialization. TODO: Move to the more ergonomic LazyLock when MSRV is 1.80
+    /// In fips mode we expect someone to have done this already.
+    #[cfg(any(not(feature = "fips"), coverage))]
```

Comment:
```suggestion
    #[cfg(any(not(feature = "fips"), coverage))]
```

Can't there be a default here? instead of doing `not feature`?

Reply:
how would we write that in a cfg line? we want this to be run in  both coverage checks or for a non-fips build